orm/src/main/java/org/hibernate/tool/internal/reveng/dialect/SQLServerMetaDataDialect.java

```diff
@@ -39,7 +39,7 @@
 					protected Map<String, Object> convertRow(ResultSet tableRs) throws SQLException {
 						element.clear();
 						element.put("TABLE_NAME", tableRs.getString("table_name"));
-						element.put("TABLE_SCHEM", sc);
+						element.put("TABLE_SCHEMA", sc);
```

Comment:
I see this TABLE_SCHEM in all the dialects, is this ok ? It looks like a typo to me

Reply:
Actually there's a lot of _SCHEM ending strings. I really think it's a typo copied/replaced everywhere